app/views/plans/genesis.html.haml

```diff
@@ -0,0 +1,9 @@
+= form_tag({}, method: "get") do
+  = label_tag "days", "Plans created in the past:"
+  = number_field_tag "days", @days, min: 1, max: 100
+  = label_tag "days", "days"
```

Comment:
This should probably have a "Search" or "Go" button in case people don't know to press enter